tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -79,8 +79,7 @@ static_assert(ranges::enable_borrowed_range<span<int>>);
 static_assert(ranges::enable_borrowed_range<span<int, 3>>);
 #endif // __cpp_lib_concepts
 
-// For performance, our implementation provides an additional guarantee beyond the Standard
-// that span and its iterator types are trivially copyable.
+// For performance, our implementation also guarantees that span's iterator types are trivially copyable.

```

Comment:
Alternatively add a transition comment and make it a good first issue when the paper has actually been merged?

Reply:
I'd prefer a non-transition comment - here and in `P0220R1_string_view` - that we provide this non-standard guarantee which WG21-P2251 may eventually standardize. I think that would cause less confusion in the interim and in the long term whether or not WG21 accepts P2251 into the working draft.

Reply:
So to be clear, revert this change? Or keep it with a currently half correct comment and have `string_view` have a fully correct comment?

I'm assuming reverting?

The GitHub issue for the paper is marked "tentatively-ready-for-plenary" so it's rather likely to be merged, right? That + the paper being "require what's already implemented", would it even be a big deal for the half correct comment?

> All we need to declare victory is annotation in yvals_core.h that _HAS_CXX20 directly controls implementation of P2251R1 - why not add that here?

Out of curiosity, given that this also affects a C++17 type, would that just be duplicated under `_HAS_CXX17` (1 person may find that _tremendously_ confusing as `span` - a non existent type in that mode - magically becomes trivially copyable)?

Reply:
> I'm assuming reverting?

More like revert-and-add:
```suggestion
// We provide a non-standard guarantees that span and its iterator types are trivially copyable; P2251 may eventually standardize that guarantee for span itself.
```
With something similar for `basic_string_view` in the other test file.